Engine/Alphas/DefaultAlphaHandler.cs

```diff
@@ -275,9 +191,19 @@ public virtual void Run()
                     throw;
                 }
 
-                Thread.Sleep(50);
+                Thread.Sleep(1);
```

Comment:
Curious if this made a difference?

Reply:
This was 50 before because we were just doing persistence/messaging updates, but now since we're doing scoring as well, the 50ms sleep was way too long